### ISSUE TITLE
Contradicting statements about commit message format

### ISSUE THREAD

**Issue by varesa** [nmstate/nmstate#1609]
There seem to be two sets of guidelines for contributions,

one in this repository: https://github.com/nmstate/nmstate/blob/base/CONTRIBUTING.md

and another one at nmstate.io: https://nmstate.io/devel/dev_guide.html or https://github.com/nmstate/nmstate.github.io/blob/base/devel/dev_guide.md

The file in this repository requests the following format for the commit message:

>   Summarize changes in around 50 characters or less
>
> Rest of the 
> commit message
> and misc.

While the nmstate.io dev guide wants the following:

> section: brief description
>
> Full descritption (sic) including root cause analyze, code workflow, test coverage.

The nmstate.io description seems more specific (also giving an example with `ovsdb: Preserve the NM external_ids`). Also most of the recent commits (but not all) mention sections, which makes me think that the CONTRIBUTING.md should probably be updated. 

Possibly one of them should just point to the other to prevent any future discrepancies if one gets updated but the other one is forgotten?

**Comment by ffmancera** [comment]
Yes I agree on this. I will update the one on nmstate.io to point to CONTRIBUTING.md
